full/Dockerfile

```diff
@@ -267,20 +211,17 @@ USER root
 RUN curl -o /var/lib/apt/dazzle-marks/docker.gpg -fsSL https://download.docker.com/linux/ubuntu/gpg \
     && apt-key add /var/lib/apt/dazzle-marks/docker.gpg \
     && add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable" \
-    && apt-get update \
-    && apt-get install -y docker-ce docker-ce-cli containerd.io slirp4netns docker-compose \
-    && cp /var/lib/dpkg/status /var/lib/apt/dazzle-marks/tool-docker.status \
-    && apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/*
+    && install-packages docker-ce docker-ce-cli containerd.io slirp4netns docker-compose
```

Comment:
Oops. Changing any lines in the "Docker Dazzle layer" will run into the same problem as https://github.com/gitpod-io/workspace-images/pull/331 i.e. it will upgrade the version of `docker` in `workspace-full` and thus break Docker-in-Gitpod.

(Also, even if you _don't_ change any lines here, I believe that the base image change will be enough to nuke/rebuild all the Dazzle layers anyway.)

Thus, this Pull Request most likely depends on https://github.com/gitpod-io/gitpod/issues/3051 so either we first fix that issue, or we break Docker-in-Gitpod.

Reply:
Valid point from @csweichel on Slack:

> can’t we pin Docker do a known working version?

So if you pin the `docker-ce` and `docker-ce-cli` versions to what's currently live in Gitpod (it's `19.03.14` -- see https://github.com/gitpod-io/workspace-images/pull/331#issuecomment-768968566) that should solve the Docker problem (and remove the dependency on https://github.com/gitpod-io/gitpod/issues/3051)

Reply:
done